develop/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN export PATH=$PATH:$(dirname $(${CCPREFIX}gcc -print-prog-name=cc1))
 RUN ln -s /usr/lib/$(gcc -dumpmachine)/libstdc++.so.6 /usr/lib/$(gcc -dumpmachine)/libstdc++.so
 
 # perl modules
-ADD . /client
```

Comment:
It seems that without mount folder this container is useless.

Reply:
No, no need to copy all files of repo to image.
Following Dockerfile commands require only cpanfile and dist.ini. Main app service from docker-compose.yml uses its own volume with repo.

Reply:
Your words only confirm what I said. Container is useless without volumes.